templates/kubernetes/docs/certs-and-trust.md

```diff
@@ -0,0 +1,158 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "Certificates and Trust Management"
+  description: How CDK manages PKI certificates and trust
+keywords: juju, kubernetes, cdk, security, pki, encryption, vault, ha
+tags: [operating, security]
+sidebar: k8smain-sidebar
+permalink: certs-and-trust.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+# Securing communications with PKI
```

Comment:
There is already a `<h1>` on this page above this heading - 'Certificates and Trust Management'. this is considered bad practice

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
Yeah, sorry, that is a hangover from the way the MD used to be rendered

Reply:
fixed